src/Core/Common/GetImageSize/Code.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+# Copyright NumFOCUS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import itk
+
+if len(sys.argv) != 2:
+    print('Usage: ' + sys.argv[0] + ' <inputImageFile>')
+    sys.exit(1)
+
+Dimension = 2
+
+PixelType = itk.UC
+
+inputImageFile = sys.argv[1]
+
+image = itk.imread(inputImageFile, PixelType)
+
```

Comment:
While we can keep these below, we can also add,

```
# Equivalent
size = itk.size(image)

print(size)

# Corresponds to the NumPy ndarray shape. Note that the ordering is reversed.
print(image.shape)

Reply:
Nice ! Done in f0247e0.